helm/hello-world-ui/values.yaml

```diff
@@ -19,3 +19,7 @@ ingress:
 api:
   # These may need to be overridden when deploying
   url: /hello-service
+sso:
+  acs: "https://localhost:8080"
```

Comment:
Should these defaults have the corresponding contexts in the values, i.e. `/alfresco`, `/activiti-app`, `/auth`?

Reply:
I break each URL used by the app into two parts. The variable part that depends on how you deploy each component and the constant part that it doesn't change. Example: the constant part for identity service is still hardcoded [here](https://github.com/Alfresco/alfresco-anaxes-hello-world-ui/blob/master/src/app/app.component.ts#L39). You will be able to set up the URL up until the "/auth/..." because everything else never changes. That's why I have made the table in the ReadMe. That should be read as "If component Alfresco Identity Service can be accessed at https://localhost:8080/auth you should give to the Identity Service environmental variable this value". Did you have a different view on this?

Reply:
That's fine for now, but in general we shouldn't assume the context (or first part of the path) where components are deployed, i.e. maybe an organization wants to host the Identity Service at `login.example.com` and the `authConfig.issuer` would be `login.example.com/realms/alfresco`.